automation/run-tests.sh

```diff
@@ -227,8 +229,12 @@ function prepare_network_environment {
         check_iface_exist $device
         if [ $? -eq 1 ]; then
             CREATED_INTERFACES="${CREATED_INTERFACES} ${device}"
-            exec_cmd "ip link add ${device} type veth peer name ${peer} && ip link set ${peer} up"
-	    exec_cmd "nmcli device set ${device} managed yes"
+            exec_cmd "ip netns add ${VETH_PEER_NS}"
```

Comment:
AFAICS, this is wrong because it is creating the netns for each device. It seems it does not have effects at all. IMO, this should be outside of the for loop. Thanks!